server/blob/blob-api/src/main/java/org/apache/james/blob/api/ObjectStoreException.java

```diff
@@ -0,0 +1,12 @@
+package org.apache.james.blob.api;
```

Comment:
As an ASF rule we must prepend all files with the ASF v 2 license...

You need to add it.

I would also suggest you set up your IDE templates.

Reply:
done :)